### PR TITLE
LPS-138748 Fix sort direction function

### PR DESCRIPTION
**Pull request by MariMalta**

### CODE REVIEW

**Comment by MariMalta** [comment]
ci:test:relevant

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
Hey @MariMalta, can you please check if there are broken tests related to your changes? Thx

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/apps/portal-workflow/portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/display/context/WorkflowDefinitionDisplayContext.java [hunk 1]
```diff
@@ -420,26 +416,19 @@ public String getSortingURL(HttpServletRequest httpServletRequest)
...
+				String orderByType = getOrderByType();
+
+				if (Validator.isNotNull(orderByType)) {
+					if (Objects.equals(orderByType, "asc")) {
+						return "desc";
+					}
 
-				if (Objects.equals(orderByType, "asc")) {
-					return "desc";
+					return "asc";
```
Comment (MarinhoFeliphe) [inline modules/apps/portal-workflow/portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/display/context/WorkflowDefinitionDisplayContext.java:431]: Hi, @MariMalta!

I believe this logic can be simplified.

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
ci:stop

**Comment by MariMalta** [comment]
ci:test:sf

**Comment by MariMalta** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
Just started reviewing :)

**Review by MarinhoFeliphe — APPROVED** [review]

**Comment by MarinhoFeliphe** [comment]
resent [here](https://github.com/liferay-workflow/liferay-portal/pull/621)

**Pull request closed without merging**